akka-persistence-typed/src/main/scala/akka/persistence/typed/javadsl/Effect.scala

```diff
@@ -143,7 +142,7 @@ import akka.persistence.typed.internal._
   /**
    * Run the given callback after the current Effect
    */
-  def andThen(chainedEffect: SideEffect[State]): Effect[Event, State]
+  def andThen(chainedEffect: State => Unit): Effect[Event, State]
```

Comment:
This must be wrong, it's a Scala function. Unused, so remove. The real thing is `thenRun`.

Reply:
Ups, yes. wrong indeed.